spec/commands_spec.rb

```diff
@@ -199,6 +199,9 @@ def rakefile_source
       Dir.mkdir(@plugins_path)
       Dir.chdir(@plugins_path)
       @command = Nesta::Commands::Plugin::Create.new(@name)
+      @namespace_path = @command.namespace_path(@gem_name)
+      @init_path = File.join(@plugins_path, @gem_name, 'lib', @namespace_path)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.